src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -373,8 +411,8 @@ export class DotEditContentHtmlService {
             save: (contentletEvent: any) => {
                 if (this.currentAction === DotContentletAction.ADD) {
                     this.renderAddedContentlet(contentletEvent.data);
-                } else if (this.currentAction === DotContentletAction.EDIT) {
-                    this.renderEditedContentlet(contentletEvent.data);
+                } else {
+                    this.renderEditedContentlet(this.currentContentlet || contentletEvent.data);
```

Comment:
this should be something like:

                if (this.currentAction === DotContentletAction.ADD) {
                    data = contentletEvent.data;
                } else  {
                    data = this.currentContentlet || contentletEvent.data;
                } 
                this.renderEditedContentlet(this.currentContentlet || contentletEvent.data);

Reply:
renderAddedContentlet and renderEditedContentlet are different methods.